wet-boew/geomap/geomap-en.html

```diff
@@ -0,0 +1,1557 @@
+---
+{
+	"title": "Geomap",
+	"language": "en",
+	"category": "Plugins",
+	"description": "Displays a dynamic map over which information from additional sources can be overlaid.",
+	"tag": "geomap",
+	"parentdir": "geomap",
+	"altLangPrefix": "geomap",
+	"js": ["demo/client_functions"],
+	"dateModified": "2016-04-22"
+}
```

Comment:
Please to the following to all 4 HTML files' front-matter:
- Please remove "category", "tag", and "parentDir".
- Rename "altLangPrefix" to "altLangPage" and add the language suffix and file extension, i.e.: "geomap-en.html"

```suggestion
{
	"title": "Geomap",
	"language": "en",
	"description": "Displays a dynamic map over which information from additional sources can be overlaid.",
	"altLangPage": "geomap-en.html",
	"js": ["demo/client_functions"],
	"dateModified": "2016-04-22"
}
```